summit.html

```diff
@@ -26,9 +26,9 @@
       <div>
         <h1 class="section-heading">The practicals</h1>
         <p class="section-paragraph">
-	  <ul>
-	    <li>Where: <a target="_blank" href="https://teams.microsoft.com/l/meetup-join/19%3ameeting_NGM1NDM4OGYtYzNmZi00OGY2LTlmODEtZmMyMmY0MDRkMWYy%40thread.v2/0?context=%7b%22Tid%22%3a%2292e84ceb-fbfd-47ab-be52-080c6b87953f%22%2c%22Oid%22%3a%2249725723-aa8c-4dcf-b9d0-b974e8a25702%22%7d">Microsoft Teams Meeting</a></li>
-	    <li>When : 22nd–23rd of September, 9:00–16:00 CEST</li>
+    <ul>
+      <li>Where: <a target="_blank" href="https://teams.microsoft.com/l/meetup-join/19%3ameeting_NGM1NDM4OGYtYzNmZi00OGY2LTlmODEtZmMyMmY0MDRkMWYy%40thread.v2/0?context=%7b%22Tid%22%3a%2292e84ceb-fbfd-47ab-be52-080c6b87953f%22%2c%22Oid%22%3a%2249725723-aa8c-4dcf-b9d0-b974e8a25702%22%7d">Microsoft Teams Meeting</a></li>
+      <li>When : 22nd–23rd of September, 9:00–16:00 CEST</li>
```

Comment:
Thou hath messed up the indentation.

Reply:
The indentation was mixed tabs and spaces.
I merely did "s/\t/  /g" :)